docs/deployment.md

```diff
@@ -0,0 +1,204 @@
+
+### Adding inventories for a new web instance
+
+To add inventories for an existing client, first clone the opensrp inventories repo [here](https://github.com/onaio/opensrp-inventories), then navigate to the host/client folder of choice. Also remember to clone the [playbooks](https://github.com/opensrp/playbooks/) repo in the same location as the opensrp inventories and follow the setup instructions as described [here](https://github.com/opensrp/playbooks#setup).
+
+Each host has a stage, preview and production folder. Change dir into the environment folder of choice then create a `vars.yml` file with a set of variables used by the 4 ansible roles listed below that are used when running the playbook command:
+
+- certbot - Installs certbot that is used to create website SSL certificates
+- react - A role used to install react apps
+- express - A role that installs and configures an express app
+- nginx - Installs and configures Nginx
+
+i.e if the client is `wellness-pass` and we want to deploy a staging site, we create the vars.yml file under `wellness-pass/stage/group_vars/web` directory.
+
+Ansible roles usually offer a level of abstraction useful for organizing playbooks.
+
+Below is a list of vars to be added in the host `vars.yml` file to be used by the above roles:
+
+```
+# General
+ansible_ssh_user: "ubuntu"
+ansible_ssh_host: <insert site url>
+
+# keycloak
+keycloak_url: <insert keycloak url>
+keycloak_realm: <insert keycloak realm>
+
+# OpenSRP
+opensrp_server_url: <insert opensrp server base url>
+opensrp_client_id: <insert oauth client id>
+
+# Web application
+app_env: <insert environment i.e preview>
+app_title: <add app title>
+
+# react
+web_react_transpile_packages: true
+web_react_remote_js_build: false # true when using admin host
+web_react_system_user: "web"
+web_react_system_user_home: "/home/{{ web_react_system_user }}"
+web_react_codebase_path: "{{ web_react_system_user_home }}/react"
+web_react_public_dir: "{{ web_react_codebase_path }}/app/build"
+web_react_local_app_path: "{{ react_local_checkout_path }}/app"
+web_react_git_url: "https://github.com/opensrp/web.git"
+web_react_git_version: "v1.1.0" // update to use latest tag
+web_express_system_user: "{{ web_react_system_user }}"
+web_express_system_user_home: "/home/{{ web_react_system_user_home }}"
+web_react_app_settings:
+  SKIP_PREFLIGHT_CHECK: "true"
+  REACT_APP_ENV: "{{ app_env }}"
+  REACT_APP_DOMAIN_NAME: "https://{{ deployed_site_name }}"
+  REACT_APP_WEBSITE_NAME: "{{ app_title }}"
+  // other env vars
+
+# express
+web_express_git_url: "https://github.com/onaio/express-server.git"
+web_express_git_version: "v1.1.0" // use latest tag
+web_express_git_key:
+web_express_app_name: "{{ web_express_system_user }}"
+web_express_codebase_path: "{{ web_express_system_user_home }}/express"
+web_express_app_path: "{{ express_checkout_path }}"
+web_express_log_path: "/var/log/{{ express_app_name }}"
+web_pm2_express_service_name: opensrp_wellnesspass_web
```

Comment:
change this variable value to `opensrp_web_stage`

Reply:
addressed